examples/camera/camera_coloured_ball_detect.py

```diff
@@ -8,23 +8,29 @@ def process_frame(frame):
     detected_balls = ball_detector.detect(frame, color=["red", "green", "blue"])
 
     red_ball = detected_balls.red
-    if red_ball.is_valid():
+    if red_ball.found:
         print(f'Red ball center: {red_ball.center}')
+        print(f'Red ball radius: {red_ball.radius}')
+        print(f'Red ball angle: {red_ball.angle}\n')
```

Comment:
Newline would be better as `print()` n next line.

Reply:
you mean just add `print()` below?

Reply:
```
print(f'Red ball angle: {red_ball.angle}')
print()
```